website/addons/dryad/templates/log_templates.mako

```diff
@@ -0,0 +1,7 @@
+<script type="text/html" id="dryad_doi_set">
```

Comment:
This file doesn't need to exist -- it should now be handled by the `.json` logs.